source/monkeyPatches/__init__.py

```diff
@@ -0,0 +1,14 @@
+# A part of NonVisual Desktop Access (NVDA)

```

Comment:
Since you're grouping all monkey patches in one package can our monkey patches for `wx` be moved to it  from `gui.initialize`?

Reply:
I'm not sure of the implications of importing wx so early in the process. Perhaps I'll do that in a separate function and have it called from `gui.initialize`